WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -37,6 +38,8 @@ enum FeatureFlag: Int, CaseIterable {
             return false
         case .newTabIcons:
             return true
+        case .useURLSession:
+            return BuildConfiguration.current != .appStore
```

Comment:
Did you consider making this a remote flag?

Reply:
See https://github.com/woocommerce/woocommerce-ios/pull/11927#discussion_r1482119827